assembl/graphql/document.py

```diff
@@ -56,9 +60,14 @@ def mutate(root, args, context, info):
 
         uploaded_file = args.get('file')
         if uploaded_file is not None:
+
             # Because the server is on GNU/Linux, os.path.basename will only work
             # with path using "/". Using ntpath works for both Linux and Windows path
             filename = ntpath.basename(context.POST[uploaded_file].filename)
+            extension = filename.split('.')[~0]
+            if extension not in get_config()['allowed_extensions']:
+                error = _('Sorry, this file type is not allowed.')
```

Comment:
Please also add an info log here that an extension of type X was added. We can aggregate these logs in future to track what types people are adding that perhaps we should incorporate by default. Also, put the current extension type in the error sent back to the client. The client might not even understand what an extension type is.

Reply:
Done.